group.go

```diff
@@ -15,7 +15,7 @@ type Group struct {
 	GroupId      string                 `json:"groupId"`
 	Timestamp    time.Time              `json:"timestamp,omitempty"`
 	Context      *Context               `json:"context,omitempty"`
-	Traits       map[string]interface{} `json:"traits,omitempty"`
+	Traits       Traits                 `json:"traits,omitempty"`
```

Comment:
Could this be left `map[string]interface{}`? That way users can opt in tot type safety with the `Traits` if they want, but ignore if they'd prefer map.